src/mame/konami/hornet.cpp

```diff
@@ -641,8 +678,8 @@ void hornet_state::sysreg_w(offs_t offset, uint8_t data)
 				m_adc12138->sclk_w(BIT(data, 0));
 			}
 
-			bool sndres = (data & 0x80) ? true : false;
-			m_audiocpu->set_input_line(INPUT_LINE_RESET, (sndres) ? CLEAR_LINE : ASSERT_LINE);
+			bool const sndres = BIT(data, 7) ? true : false;
+			m_audiocpu->set_input_line(INPUT_LINE_RESET, sndres ? CLEAR_LINE : ASSERT_LINE);
```

Comment:
You don’t really need the conditional operator here – `bool const sndres = BIT(data, 7);` should work.